fdbclient/DatabaseContext.h

```diff
@@ -611,6 +611,14 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 	// Cache of the latest commit versions of storage servers.
 	VersionVector ssVersionVectorCache;
 
+	// Introduced mainly to optimize out the version vector related code (on the client side)
+	// when the version vector feature is disabled (on the server side).
+	// @param ssVersionVectorDelta version vector changes sent by GRV proxy
+	inline bool versionVectorCacheActive(const VersionVector& ssVersionVectorDelta) {
```

Comment:
it looks like the version vector constructor initializes `maxVersion` to `invalidVersion` for `ssVersionVectorCache`.  From that point forward, the PR makes things such that we don't allow `ssVersionVectorCache` to be modified with `clear()` or `applyDelta()` if `maxVersion` is set to `invalidVersion`.  

So `ssVersionVectorCache` will never be set? I may be miss-understanding the code.. !